experimental/framework/data-objects/src/signalManager/README.md

```diff
@@ -0,0 +1,91 @@
+**_Documentation is a work-in-progress_**
+
+## SignalManager
+The `SignalManager` is a DataObject that can be used to communicate transient data via signals.
+
+### Usage
+`SignalManager` is the most appropriate tool to share transient data using Fluid.  Transient data refers to short-lived and impermanent information that is not required to be persisted.
+
+User presence scenarios are key examples of situations where sending transient data is of interest, as users are required to tell other users their own information and their past data is mostly irrelavant. Using `SignalManager` over other distributed data strucurtes in these scenarios is beneficial, as it's usage does not result in the storage of data that is not useful in the long-term.
+
+
+### Creation
+Just like with DDSes, the `FluidContainer` provides a container schema where you can include `SignalManager` as a shared object you would like to load.
+
+Here is a look at how you would go about loading `SignalManager` as part of the initial objects of the container:
+
+```typescript
```

Comment:
Overall this section has good documentation of creating objects using a `ContainerSchema`.  However, it might be better to just show an appropriate ContainerSchema once and then link to docs\content\docs\build\data-modeling.md for the full explanation of the flow, so that this documentation can be more concise and SignalManager-specific.